Cargo.toml

```diff
@@ -19,21 +19,22 @@ categories = ["database"]
 license = "MIT"
 
 [workspace.dependencies]
-scylla = "0.11.1" # This crate is specifically build for this version
-once_cell = "1"
-heck = "0.4"
-tokio = { version = "1", features = ["time", "rt-multi-thread", "io-util"] }
-thiserror = "1"
-futures-util = "0.3"
-syn = { version = "2", features = ["full", "extra-traits"] }
-quote = "1"
-proc-macro2 = "1"
-serde = { version = "1", features = ["derive"] }
-serde_json = "1"
+scylla = "0.14.0" # This crate is specifically build for this version
```

Comment:
This specific version dependency is fine, for the others, please remove the patch version :) so x.y and not x.y.z